src/data/about.md

```diff
@@ -1,16 +1,16 @@
-JumboCode is an organization dedicated to helping non-profits by providing them free, custom, technology to aid their services. JumboCode is based out of Tufts University, and works primarily with non-profits in the greater Boston area. JumboCode was founded in 2015, and is 100% run by volunteers.
+JumboCode is an organization dedicated to helping non-profits by providing them free, custom, technology to aid their services. JumboCode is based out of Tufts University and works primarily with non-profits in the greater Boston area. The club was founded in 2015 and is 100% run by student volunteers.
 
 ### Our Goals
 
-We have two main goals encompassing our work:
+There are two main goals that encompass our work:
 
 1. To support local communities in social and civic endeavorments
 2. To empower students with the technological skills to be impactful
 
 ### Our Process
 
-JumboCode projects run throughout the academic school year. In the summer, we work with non-profits to gauge their needs, and workshop potential technological solutions. Well fitting projects are assigned a project lead, who then begins working more closely with their client to plan out their project. Near the end of the summer, project leads begin developing architecture bases for their code bases.
+JumboCode projects run throughout the academic school year. In the summer, we work with non-profit organizations to gauge their needs, and brainstorm potential technological solutions. Well-fitting projects are assigned a project lead, who then begins working more closely with their client to plan out the project. Near the end of the summer, project leads begin developing architecture bases for their code bases.
```

Comment:
Also not your doing, but "well-fitting projects" feels weird for me

Reply:
Ya I thought the same thing... Will find a change in a sec